packages/flutter/test/cupertino/text_selection_test.dart

```diff
@@ -80,6 +83,33 @@ void main() {
     return button.pressedOpacity! < 1.0;
   }
 
+  // Returns the first RenderEditable.
+  RenderEditable findRenderEditable(WidgetTester tester) {
```

Comment:
At some point I tried to move stuff like this into utils files so that they could be shared.  Could you use this one here?
https://github.com/flutter/flutter/blob/e9c1c1627adc424cf3dc78ef3994e457e041503f/packages/flutter/test/widgets/editable_text_utils.dart#L11

Reply:
sure thing!